programs/cinnamon.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "cinnamon",
+    "files": [
+        {
+            "path": "$HOME/.cinnamon/spices.cache/",
+            "movable": true,
+            "help": "Supported since _5.6.0_.\n\nYou can either delete or move the folder to _XDG_CONFIG_HOME/cinnamon/spices_.\n"
```

Comment:
Damn, this should have been XDG_CACHE_HOME, idk how I missed that, apologies

Reply:
addressed via #198